### PR TITLE
dotnet9

### PR DESCRIPTION
**Pull request by marcin-krystianc**
- updating to dotnet9
- working around nuget auditing errors

### CODE REVIEW

File: scripts/perftests/testCases/OrchardCore-5dbd92c.ps1 [hunk 1]
```diff
@@ -20,7 +20,8 @@ if ($globalJsonPath) {Remove-Item "$sourcePath\$globalJsonPath"}
 
 #########################################################
 # https://github.com/G-Research/NuPerfMonitor/issues/8
-Set-Content -Path "$sourcePath\MSBuild.rsp" -Value "-p:EnableSdkContainerSupport=false"
```
Comment (mfkl) [inline scripts/perftests/testCases/OrchardCore-5dbd92c.ps1:23]: is `-p:EnableSdkContainerSupport=false` not needed anymore? If it's not needed, I guess the comment above can be removed.

File: scripts/perftests/testCases/OrchardCore-5dbd92c.ps1 [hunk 2]
```diff
@@ -20,7 +20,8 @@ if ($globalJsonPath) {Remove-Item "$sourcePath\$globalJsonPath"}
...
+# Workaround for errors due to a vulnerability scanning (https://github.com/NuGet/Home/blob/dev/proposed/2022/vulnerabilities-in-restore.md)
+New-Item "$sourcePath\Directory.Build.rsp" -Value "/p:NuGetAudit=disable"
```
Comment (mfkl) [inline scripts/perftests/testCases/OrchardCore-5dbd92c.ps1:24]: `NuGetAudit=false` ?

Comment (marcin-krystianc) [reply scripts/perftests/testCases/OrchardCore-5dbd92c.ps1:23]: Nah, it seems it needs to be `disable`, see https://github.com/NuGet/Home/blob/dev/proposed/2022/vulnerabilities-in-restore.md#enabling-vulnerability-auditing

Comment (mfkl) [reply scripts/perftests/testCases/OrchardCore-5dbd92c.ps1:23]: saw this doc https://learn.microsoft.com/en-us/nuget/concepts/auditing-packages#disabling-security-auditing but alright 👍

**Review by mfkl — APPROVED** [review]

**Pull request merged**